### PR TITLE
Case Study Memory leak using large network images.

### PR DESCRIPTION
**Pull request by terrylucas**
Consider loading images in another isolate to fix UI stall e.g., downloading images on the UI thread.

However, this is a very simple app showing simple fixes using the imageCache to not explode memory with large images downloaded over the network.

### CODE REVIEW

File: case_study/memory_leaks/images_1/lib/main.dart [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+            // action button
+            icon: const Icon(Icons.add),
+            onPressed: () {
+              _jumpNewPage();
+            },
+          )
+        ],
+        bottom: TabBar(
+            controller: _tabController,
+            tabs: tabs.map((e) => Tab(text: e)).toList()),
```
Comment (jacob314) [inline case_study/memory_leaks/images_1/lib/main.dart:102]: nit: add trailing comma after toList()

File: case_study/memory_leaks/images_1/lib/main.dart [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+      ),
+      body: listView(),
+    );
+
+    return scaffold;
+  }
+
+  /// *********************************************************************************
+  /// ***** ListView outside of TabBarView, ListView w/o width/height smaller.    *****
```
Comment (jacob314) [inline case_study/memory_leaks/images_1/lib/main.dart:111]: Also consider removing the print statements as users won't have the benefit of them in typical apps they are debugging.

Comment (terrylucas) [reply case_study/memory_leaks/images_1/lib/main.dart:111]: I meant to change these to developer.log.

File: case_study/memory_leaks/images_1/lib/main.dart (hunk 1 above)
Comment (terrylucas) [reply case_study/memory_leaks/images_1/lib/main.dart:102]: Done.

**Pull request merged**